conf.py

```diff
@@ -157,11 +161,38 @@
     'custom.css',
     'header.css',
     'github_issue_links.css',
-    'furo_colors.css'
+    'furo_colors.css',
+    'footer.css'
 ]
 html_css_files.extend(custom_html_css_files)
 
-html_js_files = ['header-nav.js']
+html_js_files = ['header-nav.js', 'footer.js']
 if 'github_issues' in html_context and html_context['github_issues'] and not disable_feedback_button:
     html_js_files.append('github_issue_links.js')
 html_js_files.extend(custom_html_js_files)
+
+#############################################################
+# Display the contributors
+
+def get_contributors_for_file(github_url, github_folder, pagename, page_source_suffix, display_contributors_since=None):
+    filename = f"{pagename}{page_source_suffix}"
+    paths=html_context['github_folder'][1:] + filename
+    repo = Repo(".") 
+    since = display_contributors_since if display_contributors_since and display_contributors_since.strip() else None
+
+    commits = repo.iter_commits('--all', paths=paths, since=since)
```

Comment:
```suggestion
    commits = repo.iter_commits(paths=paths, since=since)
```

We need to get rid of the `--all` here - I honestly don't really understand [what the option does](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---all), but for bigger documentation sets it results in some commits being included that shouldn't be.

Reply:
It is used to ensure that all the commit from all the branches are added.